hunts/models.py

```diff
@@ -1,18 +1,52 @@
 from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
 from django.db.models import Q
 from django.shortcuts import get_object_or_404
 from django.template.defaultfilters import slugify
 from django.utils import timezone
 from puzzles.models import Puzzle
 
+from google_api_lib import sync_tasks
+
+
+class GoogleInfo(models.Model):
+    hunt_folder_id = models.CharField(max_length=44)
+    autogenerated_sheets_folder_id = models.CharField(
+        max_length=44, editable=False, blank=True
+    )
+    human_files_folder_id = models.CharField(max_length=44, editable=False, blank=True)
+    sheets_template_file_id = models.CharField(max_length=44, blank=True)
+    whitelisted_emails = ArrayField(models.EmailField(), editable=False, blank=True)
+
+    def save(self, *args, **kwargs):
+        if not self.autogenerated_sheets_folder_id:
+            self.autogenerated_sheets_folder_id = self.hunt_folder_id
+
+        if not self.human_files_folder_id:
+            self.human_files_folder_id = self.hunt_folder_id
+
+        if not self.whitelisted_emails:
+            self.whitelisted_emails = sync_tasks.get_file_user_emails(
+                self.hunt_folder_id
+            )
+
+        super(GoogleInfo, self).save(*args, **kwargs)
+
 
 class Hunt(models.Model):
     name = models.CharField(max_length=128)
     url = models.URLField()
     created_on = models.DateTimeField(auto_now_add=True)
     start_time = models.DateTimeField(default=None, blank=True, null=True)
     end_time = models.DateTimeField(default=None, blank=True, null=True)
+    google_ids = models.OneToOneField(
```

Comment:
Might rename this to `google_config` or something, I thought `ids` meant users at first